client/packages/purchasing/src/purchase_order/api/hooks/usePurchaseOrderList.ts

```diff
@@ -61,7 +55,7 @@ export const usePurchaseOrderList = (queryParams?: ListParams) => {
       storeId,
       first: first,
       offset: offset,
-      key: sortFieldMap[sortBy.key] ?? PurchaseOrderSortFieldInput.Status,
+      key: (sortBy.key as PurchaseOrderSortFieldInput) ?? 'number',
```

Comment:
No idea why we'd want "status" as the default sort.

Reply:
It's possible it was requested, but created_datetime sound more normal...